client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/index.tsx

```diff
@@ -92,7 +94,8 @@ const LinkInBioSetup: Step = function LinkInBioSetup( { navigation } ) {
 		}
 
 		if ( siteTitle.trim().length ) {
-			submit?.( { siteTitle, tagline } );
+			const backToStep = urlQueryParams.get( 'backToStep' );
+			submit?.( { siteTitle, tagline, backToStep } );
```

Comment:
do we need to check if backToStep is undefined here? does it work if it's undefined but we still pass it to the function?

Reply:
@Sharpirate - I believe this is OK. I had checked the same thing -> If the query param is not set, this returns null, not undefined. So if not defined, we're be passing a null value, and that works OK downstream for the needed conditional check.